corehq/apps/export/views/utils.py

```diff
@@ -310,7 +310,7 @@ class DailySavedExportPaywall(BaseProjectDataView):
 
 
 class DashboardFeedPaywall(BaseProjectDataView):
-    urlname = 'dashbaord_feeds_paywall'
+    urlname = 'dashboard_feeds_paywall'
```

Comment:
nice catch. If you haven't already, it would be good to take a look at where this url is being used to make sure the bad spelling isn't expected somewhere else. All tests are passing so I would guess this is a good fix, but worth having a look to be sure.

Reply:
I didn't find the misspelling anywhere else in the codebase, and looks like the actual url is already correct at domain/data/export/dashboard_feed/paywall/